pip_audit/_cli.py

```diff
@@ -159,6 +160,12 @@ def _parser() -> argparse.ArgumentParser:
         dest="requirements",
         help="audit the given requirements file; this option can be used multiple times",
     )
+    dep_source_args.add_argument(
+        "-p",
+        "--pyproject",
+        type=argparse.FileType("r"),
+        help="audit the given `pyproject.toml` file",
+    )
```

Comment:
See my comment here: https://github.com/trailofbits/pip-audit/issues/81#issuecomment-1032988316

We shouldn't need a `-p` or `--pyproject` flag, we should auto-detect the presence of a `pyproject.toml` file when auditing a local source tree.

Reply:
Agreed; `pip-audit .` should intelligently select either `setup.py` (not implemented yet) or `pyproject.toml`, with a preference for the latter if it has dependencies listed.

Reply:
My bad, I missed that comment. Should be an easy fix though.